lib/rubocop/cop/style/space_after_colon.rb

```diff
@@ -12,21 +12,35 @@ class SpaceAfterColon < Cop
 
         def on_pair(node)
           oper = node.loc.operator
-          return unless oper.is?(':') && followed_by_space?(oper)
 
-          add_offense(oper, oper)
+          return unless oper.is?(':')
+
+          add_offence_if_colon_not_followed_by_space(oper)
```

Comment:
I remember my own struggles with "color" vs. "colour" in the early days of the internet (before auto-complete was even conceived.) :sweat_smile: